tests/test_domain_models.py

```
@@ -0,0 +1,121 @@
+import pytest
+from decimal import Decimal
+from datetime import datetime
+from src.domain.models import (
+    Coordinates, Location, Sensor, Measurement,
+    ParameterType, MeasurementUnit
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)